### DISCUSSION TITLE
It's crash after few minutes. when I'm using useEffect without dependencies

### DISCUSSION THREAD

**Discussion by amitCodemonk** [codesandbox/sandpack#999] [Q&A]
when I am using useEffect without dependencies , and state will change each time then , It will crash page is not reload and changes will reflect after some time if it run continuously 5 to 6 minute then changes will not reflect .it takes lot of time.

![image](https://github.com/codesandbox/sandpack/assets/144655303/6d1bf037-b570-41f2-a3d4-05f0b9ba52e6)

here is my code : 
![image](https://github.com/codesandbox/sandpack/assets/144655303/8f5d6fb8-9920-434e-a10e-20b24521f8b3)

how I stop it before it crashed..

**Comment by danilowoz** [comment]
This behavior is expected when setting a state inside of useEffect, as each state update triggers useEffect again. This is how React works and not a Sandpack issue

**Reply by amitCodemonk** [reply]
Thanks for reply, 
Is there any way if some one define state inside of useEffect , page is not crash . How can I stop the crash issue. 
I am just try to build a 2 user use same code editor at a same time . But ,if one of the user defined state inside the code editor. whole  page is  not working .it's crashed .
